src/Modules/Workspaces/Commands.Workspaces.Test/GetPowerBIWorkspaceTests.cs

```diff
@@ -48,6 +48,76 @@ public void EndToEndGetWorkspacesOrganizationScope()
             }
         }
 
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void EndToEndGetWorkspacesAllAndOrganizationScope()
```

Comment:
You should also write unit tests that mock up the SDK client and verify that the Get method was called the correct amount of times.